scripts/imgr

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+usage()
+{
+cat << EOF
+Usage: $0 IOCNAME [--hutch HUTCH] --COMMAND
+
+Script to control an ioc. 
+
+List of commands:
+$0 IOCNAME [--hutch HUTCH] --reboot soft
+$0 IOCNAME [--hutch HUTCH] --reboot hard
+$0 IOCNAME [--hutch HUTCH] --enable
+$0 IOCNAME [--hutch HUTCH] --disable
+$0 IOCNAME [--hutch HUTCH] --upgrade RELEASE_DIR
+$0 IOCNAME [--hutch HUTCH] --move HOST
+$0 IOCNAME [--hutch HUTCH] --move HOST:PORT
+$0 [--hutch HUTCH] --list [--host HOST] [--enabled_only|--disabled_only]
+ 
+EOF
+}
+
+if [ $# -lt 1 ]; then
+   echo 'need arguments' >&2
+   usage
+   exit 0
+fi
+
+
+if [[ ($1 == "--help") || ($1 == "-h") ]]; then
+   usage
+   exit 0
+fi
+
+ulimit -c unlimited 
+
+source /reg/g/pcds/setup/pathmunge.sh
+ldpathmunge /usr/local/lib
+
+DIR=$(readlink -f $(dirname "${BASH_SOURCE[0]}"))
+PATH=$PATH:$DIR
+pathmunge $DIR
+
+/reg/g/pcds/pyps/apps/iocmanager/latest/imgr $@
```

Comment:
Why is that?

Reply:
Otherwise, what's the point of having different versions for each hutch? Maybe a new release will break things in one hutch so they will want to stick with an older version.